browser/create-template/templates/nextjs-site/src/components/Image.tsx

```diff
@@ -0,0 +1,242 @@
+import { store } from '@/app/store';
+import { type Resource, type Server, unknownSubject, server } from '@tomic/lib';
+import React from 'react';
+import NextImage from 'next/image';
+
```

Comment:
Almost everything here should be imported from `@tomic/react`, otherwise this creates a lot of maintenance work.